test/e2e/scaffold/scaffold.go

```diff
@@ -56,16 +56,17 @@ type Options struct {
 }
 
 type Scaffold struct {
-	opts              *Options
-	kubectlOptions    *k8s.KubectlOptions
-	namespace         string
-	t                 testing.TestingT
-	nodes             []corev1.Node
-	etcdService       *corev1.Service
-	apisixService     *corev1.Service
-	httpbinDeployment *appsv1.Deployment
-	httpbinService    *corev1.Service
-	finializers       []func()
+	opts                   *Options
+	kubectlOptions         *k8s.KubectlOptions
+	namespace              string
+	t                      testing.TestingT
+	nodes                  []corev1.Node
+	etcdService            *corev1.Service
+	apisixService          *corev1.Service
+	httpbinDeployment      *appsv1.Deployment
+	httpbinService     *corev1.Service
```

Comment:
The fields are not aligned.

Reply:
It has been fixed, goland's gofmt occasionally does not work, which is a bit strange.